NSURLSessionCancellation/ViewController.swift

```diff
@@ -27,9 +27,9 @@ class ViewController: UIViewController {
 
         // Download data
 
-        session.dataTask(with: url, completionHandler: { (data, response, error) in
+        session.dataTask(with: url) { (data, response, error) in
```

Comment:
You don't need parentheses around the parameters (which mays the RW Style Guide examples).